yarn.lock

```diff
@@ -48,6 +48,21 @@
     ui-router-visualizer "^4.0.0"
     ui-select "^0.19.6"
 
+"@spinnaker/styleguide@^1.0.0":
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/@spinnaker/styleguide/-/styleguide-1.0.0.tgz#8662cf4778e0292540d389c5caa6e9b94536e9a0"
+  dependencies:
+    autoprefixer "^8.0.0"
+    cssnano "^3.10.0"
+    gulp "^3.9.1"
+    gulp-concat "^2.6.1"
+    gulp-postcss "^7.0.1"
+    postcss-css-variables "^0.8.0"
+    postcss-export-vars "^0.2.0"
+    postcss-extend "^1.0.5"
+    postcss-import "^11.1.0"
+    postcss-style-guide "^0.14.0"
```

Comment:
any dependencies we can take out of deck now that all the styleguide stuff is in it's own repo? Maybe some of the postcss stuff?

Reply:
Sorry, I forgot. I will remove all of them.

Reply:
done and done.